### ISSUE TITLE
Table upgrade

### ISSUE THREAD

**Issue by deboer-tim** [icpctools/team-view#122]
The tables in the problem and team pages don't match; they're missing correct styles and things like aria-roles. They don't resize properly, and adding the optional reactions column was awkward (for everyone who doesn't have reactions there's an empty column space).
